pkg/auth/providers/oidc/oidc_provider_test.go

```diff
@@ -427,6 +434,100 @@ func TestGetClaimInfoFromToken(t *testing.T) {
 	}
 }
 
+func TestGetGroupsFromClaimInfo(t *testing.T) {
+	type fields struct {
+		Name        string
+		Type        string
+		CTX         context.Context
+		AuthConfigs v3.AuthConfigInterface
+		Secrets     wcorev1.SecretController
+		UserMGR     user.Manager
+		TokenMGR    tokenManager
+	}
```

Comment:
I don't think these fields need to be exported, I would put them lowercase.

The `CTX` caught my eye. :smile: